src/mono/wasm/build/WasmApp.targets

```diff
@@ -327,6 +328,7 @@
 
   <Target Name="_GetWasmGenerateAppBundleDependencies">
     <PropertyGroup>
+      <HybridGlobalization Condition="'$(InvariantGlobalization)' == 'true'">false</HybridGlobalization>
```

Comment:
This will be only needed if both `HybridGlobalization` and `InvariantGlobalization` are true but there should not be this kind of scenario. I guess we need to have some checks before reaching this point and letting the users know that `HybridGlobalization` and `InvariantGlobalization` can't be both true.

Reply:
We don't have any mechanism preventing them both to be true. You are right, it will be better if I add a warning there as well, silently switching of the property is not the best option.